CONTRIBUTING.md

```diff
@@ -66,10 +66,12 @@ Assuming you are using standard tools:
 * Fork Sopel's repository, and clone your fork locally.
 * Activate your virtualenv, and `cd` into your clone's folder.
 * Ensure you have the latest version of `pip`, and install `wheel`.
-* Install Sopel from source as an editable install using the `dev` extra.
+* Install Sopel from source as an editable install, and install the dev
+  depencendies using the `requirements.txt` file:
```

Comment:
```suggestion
  dependencies using the `requirements.txt` file:
```

I also think we should not use the de facto standard name "requirements.txt" for non-runtime dependencies. The old "dev-requirements.txt" was fine, but something shorter would also work. "devdeps.txt" (or with a hyphen) maybe?